lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -0,0 +1,123 @@
+            accountPreferencesModel.url = options['preferences_api_url'];
+
+            var accountSettingsModel = new AccountSettingsModel({
+                'default_public_account_fields': options['default_public_account_fields']
+            });
+            accountSettingsModel.url = options['accounts_api_url'];
+
+            var editable = options['own_profile'] ? 'toggle' : 'never';
+
+            var accountPrivacyFieldView = new AccountSettingsFieldViews.AccountPrivacyFieldView({
+                model: accountPreferencesModel,
+                required: true,
+                editable: 'always',
+                showMessages: false,
+                title: gettext('edX learners can see my:'),
+                valueAttribute: "account_privacy",
+                options: [
+                    ['private', gettext('Limited Profile')],
+                    ['all_users', gettext('Full Profile')]
+                ],
+                helpMessage: '',
+                accountSettingsPageUrl: options['account_settings_page_url']
+            });
+            var usernameFieldView = new AccountSettingsFieldViews.ReadonlyFieldView({
+                    model: accountSettingsModel,
+                    valueAttribute: "username",
+                    helpMessage: ""
+            });
+            var sectionOneFieldViews = [
+                usernameFieldView,
+
+                new AccountSettingsFieldViews.DropdownFieldView({
+                    model: accountSettingsModel,
+                    required: false,
+                    editable: editable,
+                    showMessages: false,
+                    iconName: 'fa-map-marker',
+                    placeholderValue: gettext('Add country'),
+                    valueAttribute: "country",
+                    options: options['country_options'],
+                    helpMessage: ''
+                }),
+
+                new AccountSettingsFieldViews.DropdownFieldView({
+                    model: accountSettingsModel,
+                    required: false,
+                    editable: editable,
+                    showMessages: false,
+                    iconName: 'fa-comment fa-flip-horizontal',
+                    placeholderValue: gettext('Add language'),
+                    valueAttribute: "language",
+                    options: options['language_options'],
+                    helpMessage: '',
+                })
+            ];
+
+            var sectionTwoFieldViews = [
+                new AccountSettingsFieldViews.TextareaFieldView({
+                    model: accountSettingsModel,
+                    editable: editable,
+                    showMessages: false,
+                    title: gettext('About me'),
+                    placeholderValue: gettext("Tell other edX learners a little about yourself, where you're from, what your interests are, why you joined edX, what you hope to learn..."),
```

Comment:
Tell other edX learners a little about yourself: where you live, what your interests are, why you’re taking courses on edX, or what you hope to learn.